v23.1/monitoring-and-alerting.md

```diff
@@ -7,31 +7,58 @@ docs_area: manage
+
+- [Monitor CockroachDB with Prometheus](monitor-cockroachdb-with-prometheus.html)
+- [Third-party Monitoring Tools](third-party-monitoring-tools.html)
 
 {% include {{ page.version.version }}/prod-deployment/cluster-unavailable-monitoring.md %}
 
-## Monitoring tools
+<a id="monitoring-tools"></a>
+
+## Built-in monitoring tools
+
+CockroachDB includes several tools to help you monitor your cluster's workloads and performance.
 
 {{site.data.alerts.callout_danger}}
-If a cluster becomes unavailable, most of the monitoring tools in the following sections become unavailable. In that case, Cockroach Labs recommends that you use the [Prometheus endpoint](#prometheus-endpoint) or consult the [cluster logs](logging-overview.html).
+If a cluster becomes unavailable, most of the monitoring tools in the following sections become unavailable. In that case, Cockroach Labs recommends that you consult the [cluster logs](logging-overview.html). To maintain access to a cluster's historical metrics when the cluster is unavailable, configure a [third-party monitoring tool](third-party-monitoring-tools.html) like Prometheus or Datadog to collect metrics periodically from the [Prometheus endpoint](#prometheus-endpoint). The metrics are stored outside the cluster, and can be used to help troubleshoot what led up to an outage.
 {{site.data.alerts.end}}
 
 ### DB Console
 
-The [DB Console](ui-overview.html) displays essential metrics about a cluster's health, such as node status, number of unavailable ranges, and queries per second and service latency across the cluster. This tool is designed to help you optimize cluster performance and troubleshoot issues. The DB Console is accessible from every node at `http://<host>:<http-port>`, or `http://<host>:8080` by default.
+The [DB Console](ui-overview.html) collects time-series cluster metrics and displays basic information about a cluster's health, such as node status, number of unavailable ranges, and queries per second and service latency across the cluster. This tool is designed to help you optimize cluster performance and troubleshoot issues. The DB Console is accessible from every node at `http://<host>:<http-port>`, or `http://<host>:8080` by default.
+
+The DB Console automatically runs in the cluster. By default it collects time-series metrics every 10 minutes, retains records for an hour, and provides basic reporting and alerting. This simplifies monitoring and observability, but comes with some drawbacks. Some organizations require more granular collection and retention of metrics, but do not wish to impose additional workload on the cluster to process and store more metrics. Also, if the cluster is unavailable, the DB Console cannot show the cluster's performance leading up to the outage.
```

Comment:
+1 that only SQL Activity data is the main request, not metrics generally

The closest thing to this I've heard is customers wanting more granular per-object metrics (like per application, per database, per table, etc) but ack that these don't apply in the context of this doc.

Reply:
@kevin-v-ngo The 10m false assertion came from when I was thinking that the relevant settings were these, rather than the `timeseries` cluster settings. https://www.cockroachlabs.com/docs/stable/cluster-settings.html#setting-sql-stats-flush-interval.